distribution_circuits_sale/models/sale.py

```diff
@@ -128,6 +128,12 @@ def _compute_amount_due(self):
             for invoice in open_invoices:
                 amount_total += invoice.residual
             
+            # draft invoices are not taken into account in the partner credit
+            draft_invoices =  invoice_obj.search([('partner_id','=',partner.id),('state','in',['draft'])])
+            
+            for invoice in draft_invoices:
+                amount_total += invoice.amount_total
+            
```

Comment:
Not sure is the way to go. We could have invoices that would never be validated. For a functional point of view a draft invoice should not be taken into account.

In our case the invoice generated from the sale order after the delivery orders have been validated must be validated. we shouldn't find any draft invoice. 

just for info you should also fitler on type =  out_invoice to only have customer invoice has it easy you are mixing supplier invoice and customer invoice (in the case you have a supplier that is a customer) and credit note.